input/docs/cs/imagenet_classes.md

```diff
@@ -0,0 +1,22 @@
+title: Zobrazení tříd neuronových konvolučních sítí
+Description: Zobrazení tříd z konvoluční sitě Alexnet v čitelné podobě
+---
+>Tento dokument pracuje s konvoluční neuronovou sítí Alexnet.
+
+#Proč si vytvořit jednoduchý skript pro zobrazení klasifikací
+Často se stává, že je potřeba se podívat na možné třídy neuronových klasifikačních sítí, pro zjištění, zda má možnost konkrétní obrázky vůbec klasifikovat. Pro příklad, je potřeba klasifikovat obrázek želvy, v tom případě lze zobrazit následujícím kódem třídy podle abecedy a vyhledat zda se zde nachází v tomto příkladě "Turtle". Takové třídění lze provést nejen u konvoluční neuronové sítě Alexnet, ale je možné i pro další sítě př.:(googlenet,squeezenet,densenet201,resnet18...)
+
+
+# Zobrazit třídy s pozicemi použitím Alexnet
+
+```matlab
```

Comment:
"upravte názvy proměnných " - přesně tak to bylo v zadání a přesně to je potřeba..